File: warehouse-system/src/main/java/com/who/warehousesystem/dto/CheckDetailsDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.sql.Date;

@Data
@AllArgsConstructor

public class CheckDetailsDto {

    private Integer id;
    private String type;
    private Date date;
    private String notes;
}

File: WarehouseSystem/WarehouseSystem/Dto/CheckDetailsDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class CheckDetailsDto
    {
        public string id { get; set; }
        public string type { get; set; }
        public string date { get; set; }
        public string notes { get; set; }

        public CheckDetailsDto (string id,string type,string date,string notes)
        {
            this.id = id;
            this.type = type;
            this.date = date;
            this.notes = notes;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
